kotlin/internal/jvm/jvm.bzl

```diff
@@ -362,3 +364,20 @@ kt_jvm_import = rule(
     implementation = _kt_jvm_import_impl,
     provides = [JavaInfo, _KtJvmInfo],
 )
+
+kt_plugin = rule(
```

Comment:
Since kapt plugin can also be written in Kotlin, kt_compiler_plugin?

Reply:
I renamed the rule